crates/solidity/outputs/cargo/tests/src/cst_output/renderer.rs

```diff
@@ -132,8 +132,11 @@ fn render_key(cursor: &mut Cursor) -> String {
         Node::Terminal(terminal) => terminal.kind.to_string(),
     };
 
-    if let Some(label) = cursor.label() {
-        format!("({label}꞉ {kind})", label = label.as_ref().to_snake_case())
+    if cursor.has_open_parent() {
```

Comment:
I suggest keeping the original behaviour here, which (IIUC) will now print `EdgeLabel::Root`, to make sure our tests are actually testing what the user will see.

Reply:
Also, I wonder if there are any other use cases for this new `has_open_parent()` API?